packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -101,9 +105,10 @@ class DateTimeUtils {
    * @param {boolean} hasSeconds Whether or not seconds should be retrieved
    * @return {string} The time from the date and time string
    */
-  static getTime(time, hasSeconds) {
+  static getTime(time, hasSeconds, timeZone) {
     const timeFormat = hasSeconds ? 'HH:mm:ss' : 'HH:mm';
-    return DateUtil.formatISODate(time, timeFormat);
+    const momentDate = (timeZone && moment.tz.zone(timeZone) ? moment.tz(time, timeZone) : moment(time));
```

Comment:
If `timezone` is invalid, should the object be created using the guess timezone? Or better yet, just call `createSafeDate`.

Reply:
updated here 49f03acf3e9427c81be892413ff0c07bb091048d